### PR TITLE
[WIP]Tests tagging on provider side

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Add tag mapping tests
Depends on https://github.com/ManageIQ/wrapanapi/pull/266, https://github.com/ManageIQ/integration_tests/pull/7319

{{pytest: cfme/tests/cloud/test_tag_mapping.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit e9b35f3d768b32347c74552a47ea433bf2c38fc6:

`cfme/tests/cloud/test_tag_mapping.py`:
- :bangbang: Line 42:13: **E122** *continuation line missing indentation or outdented*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ca19ac5b6d70dd4a74bcf762356dcdbc118bece9:

`cfme/configure/configuration/region_settings.py`:
- :heavy_exclamation_mark: Line 1035:63: **W292** *no newline at end of file*

`cfme/tests/cloud/test_tag_mapping.py`:
- :bangbang: Line 42:13: **E122** *continuation line missing indentation or outdented*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/common/vm_views.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def data(self):
                 policy = None
 
             data_dict['policy'] = policy
-        else:
+        elif getattr(data_dict, 'quad'):
```
Comment (rlbabyuk) [inline cfme/common/vm_views.py]: A small fix here as table view doesn't have 'quad' att

File: cfme/common/vm_views.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def data(self):
                 policy = None
 
             data_dict['policy'] = policy
-        else:
+        elif 'quad' in data_dict:
```
Comment (rlbabyuk) [inline cfme/common/vm_views.py]: A small fix here as table view doesn't have 'quad' key

File: cfme/tests/configure/test_tag.py [hunk 3]
```diff
@@ -44,6 +45,30 @@ def test_tag_crud(category):
     tag.delete()
 
 
+def test_map_tagging_crud(appliance, category, soft_assert):
+    """Test Mam tagging grud with flash message assertion"""
```
Comment (sshveta) [inline cfme/tests/configure/test_tag.py]: Typo :  """Test Map tagging crud with flash message assertion"""

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
Replacing with #8543

**Pull request closed without merging**